clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/StandardGceAttributeValidator.groovy

```diff
@@ -375,6 +386,46 @@ class StandardGceAttributeValidator {
     }
   }
 
+  def validateAutoscalingPolicy(GoogleScalingPolicy policy) {
+
+    validateNotEmpty(policy, "autoscalingPolicy")
+
+    if (policy) {
+
+      policy.with {
+        validateNonNegativeLong(minNumReplicas, "autoscalingPolicy.minNumReplicas")
```

Comment:
The API accepts null fields and sets them to defaults, though currently they would never be null here (because of the defaults in GoogleScalingPolicy).